### PR TITLE
feat: Release notes for Codacy Cloud February 2023 DOCS-438

### PR DESCRIPTION
**Pull request by prcr**
Adds auto-generated release notes for Codacy Cloud February 2023

### :eyes: Live preview
https://feat-release-notes-cloud-2023-02--docs-codacy.netlify.app/release-notes/cloud/cloud-2023-02/

### :construction: To do
- [x] Add the new release notes page to `mkdocs.yml` and `docs/release-notes/index.md`
- [x] Review generated release notes
- [x] Review list of issues with missing release notes
- [x] Review links to changelogs of updated tools
- [x] Update release date and remove `TODO` comments

### CODE REVIEW

File: docs/release-notes/cloud/cloud-2023-02.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+-   https://codacy.atlassian.net/browse/TS-261
+-   https://codacy.atlassian.net/browse/TS-256
+-   https://codacy.atlassian.net/browse/TS-251
+-   https://codacy.atlassian.net/browse/TS-249
+-   https://codacy.atlassian.net/browse/TS-242
+-   https://codacy.atlassian.net/browse/TS-233
+-   https://codacy.atlassian.net/browse/TS-224
+-   https://codacy.atlassian.net/browse/TS-206
+-   https://codacy.atlassian.net/browse/TS-148
+-   https://codacy.atlassian.net/browse/IO-424
```
Comment (prcr) [inline docs/release-notes/cloud/cloud-2023-02.md:41]: @nicklem can you also check this bug please?

File: docs/release-notes/cloud/cloud-2023-02.md [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+
+Jira issues without release notes
+
+Epics:
+-   https://codacy.atlassian.net/browse/PLUTO-280
+-   https://codacy.atlassian.net/browse/PLUTO-279
+-   https://codacy.atlassian.net/browse/PLUTO-278
+-   https://codacy.atlassian.net/browse/PLUTO-244
+-   https://codacy.atlassian.net/browse/IO-437
+-   https://codacy.atlassian.net/browse/IO-289
```
[the remark this answers is not in the document]
Comment (prcr) [reply docs/release-notes/cloud/cloud-2023-02.md:26]: We set the release notes of both Epics to "Don't publish". :heavy_check_mark:

File: docs/release-notes/cloud/cloud-2023-02.md (hunk 1 above)
Comment (nicklem) [reply docs/release-notes/cloud/cloud-2023-02.md:41]: 👍 Done

File: docs/release-notes/cloud/cloud-2023-02.md [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+
+<!--TODO Check these issues manually
+
+Jira issues without release notes
+
+Epics:
+-   https://codacy.atlassian.net/browse/PLUTO-280
+-   https://codacy.atlassian.net/browse/PLUTO-279
+-   https://codacy.atlassian.net/browse/PLUTO-278
+-   https://codacy.atlassian.net/browse/PLUTO-244
```
[the remark this answers is not in the document]
Comment (claudiacarpinteiro) [reply docs/release-notes/cloud/cloud-2023-02.md:24]: > -   https://codacy.atlassian.net/browse/PLUTO-280
> -   https://codacy.atlassian.net/browse/PLUTO-279
> -   https://codacy.atlassian.net/browse/PLUTO-278

These are not to be published for now☝️. I changed the Availability field to `Don't publish this release note`. 

> -   https://codacy.atlassian.net/browse/PLUTO-244

I already updated the release note for this one.

File: docs/release-notes/cloud/cloud-2023-02.md [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+-->
+
+## Product enhancements
+
+-   Improved the reporting of analysis and coverage status to Git providers when new commits are pushed during an ongoing analysis. (IO-341)
+-   Codacy now supports [reporting a summary of the coverage metrics](../../repositories-configure/integrations/github-integration.md#enabling) directly on your pull requests. For now, this feature is only supported on GitHub Cloud. ![Coverage summary on GitHub](../images/cov-2.png) (COV-2)
+
+## Bug fixes
+
+-   Fix email notification not sent when the user sending the request doesn't have a user name (PLUTO-365)
```
Comment (claudiacarpinteiro) [inline docs/release-notes/cloud/cloud-2023-02.md:96]: I updated this release note in the [Jira issue](https://codacy.atlassian.net/browse/PLUTO-365):
"Fixed an issue that prevented organization owners from receiving the email notification with new requests to join the Codacy organization when the requester user didn’t have a name set up."

File: docs/release-notes/cloud/cloud-2023-02.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (prcr) [reply docs/release-notes/cloud/cloud-2023-02.md:24]: Thank you @claudiacarpinteiro! :bow:

File: docs/release-notes/cloud/cloud-2023-02.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+
+# Cloud February 2023
+
+These release notes are for the Codacy Cloud updates during February 2023.
+
+📢 [Visit the Codacy roadmap](https://roadmap.codacy.com) and <span class="skip-vale">let us know</span> your feedback on both new and planned product updates!
+
+## Product enhancements
+
+-   In some Enterprise plans, Codacy now automatically adds to the organization new people that commit to your private repositories and analyzes their commits, providing a more streamlined user management experience. (PLUTO-244)
```
Comment (prcr) [inline docs/release-notes/cloud/cloud-2023-02.md:18]: @AmhaBekele could you please check this release note in particular?

Comment (AmhaBekele) [reply docs/release-notes/cloud/cloud-2023-02.md:18]: LGTM

**Review by AmhaBekele — APPROVED** [review]

**Pull request merged**